_usage/locales.md

```diff
@@ -19,109 +19,117 @@ encoding. Babelfish will automatically translate from the server
+| `ci`        | Case insensitive                                       |
+| `cs`        | Case sensitive                                         |
+| `pref`      | To sort uppercase letters before lowercase letters, use a `pref` collation. If the comparison is case-sensitive, the uppercase version of a letter sorts before the lowercase version, if there is no other distinction. The ICU library supports uppercase preference with `colCaseFirst=upper`, but not for `ci_as` collations. `pref` can be applied only to `cs_as` (deterministic).  |
+
+PostgreSQL doesn't support the `LIKE` clause on non-deterministic
+collations, but Babelfish supports it for `ci_as` collations.
+Babelfish doesn't support `LIKE` on AI collations. Pattern matching
 operations on non-deterministic collations are also not supported.
 
 To establish Babelfish collation behavior, set the following
 parameters :
 
 | Parameter             | Description                                   |
 | --------------------- | --------------------------------------------- |
-| `default_locale`      | The ` default_locale ` parameter is used in combination with the collation attributes in the table above to customize collations for a specific language and region. The default value is `en-US`. The default locale applies to all Babelfish collations and to all SQL Server collations that are mapped to Babelfish collations. This parameter may be changed after initial Babelfish database creation time, but it won\'t affect the locale of existing collations.            |
-| `server_collation_name`  | The collation used as the default collation at both the server level and the database level. The default value is `sql_latin1_general_cp1_ci_as`. You can choose from the collations in the table that follows for the Collation name field when you create your PostgreSQL cluster for use with Babelfish. Don\'t modify the `server_collation_name` after the Babelfish database is created.  |
+| `server_collation_name`  | The collation used as the default collation at both the server level and the database level. The default value is `sql_latin1_general_cp1_ci_as`. When you create your PostgreSQL cluster for use with Babelfish, you can choose the &ldquo;Collation name&rdquo; from the following table. Don't modify `server_collation_name` after the Babelfish database has been created.  |
+| `default_locale`      | The `default_locale` parameter is used to customize the collation chosen with `server_collation_name` for a specific language and region. The default value is `en-US`.  This parameter is ignored if the chosen collation already implies a locale.  This parameter may be changed after initial Babelfish database creation time, but it won't affect the locale of existing collations. |
 
-Use the following collations as a server collation or an object
-collation:
+The following collations (and versions with different modifier suffixes, like `ci_as`) can be used in `server_collation_name` or for column definitions:
```

Comment:
"and versions with different modifier suffixes, like `ci_as`" - any Babelfish collation with a ci_as modifier suffix

Reply:
I guess my wording was not clear enough. I removed the parenthesis and added:
In addition to the ones listed, you can use variants with different suffixes, like `ci_as` or `ci_ai`.